lib/Controller/LoginController.php

```diff
@@ -520,6 +521,14 @@ public function code(string $state = '', string $code = '', string $scope = '',
 			$this->userSession->completeLogin($user, ['loginName' => $user->getUID(), 'password' => '']);
 			$this->userSession->createSessionToken($this->request, $user->getUID(), $user->getUID());
 			$this->userSession->createRememberMeToken($user);
+
+			// prevent password confirmation
+			if (defined(IToken::class . '::SCOPE_SKIP_PASSWORD_VALIDATION')) {
+				$token = $this->authTokenProvider->getToken($this->session->getId());
+				$token->setScope([IToken::SCOPE_SKIP_PASSWORD_VALIDATION => true]);
```

Comment:
IIRC, if another scope was set, it needs to be taken over, this would replace it.

Reply:
Thanks for the feedback.
Hmmm are you saying we should override the scope and just set `IToken::SCOPE_SKIP_PASSWORD_VALIDATION` to true?
Or are you saying we should only extend the scope and add `IToken::SCOPE_SKIP_PASSWORD_VALIDATION`?

Reply:
>  Or are you saying we should only extend the scope and add IToken::SCOPE_SKIP_PASSWORD_VALIDATION?

This one!